databricks/koalas/generic.py

```diff
@@ -77,6 +77,42 @@ def compute(self):
         """Alias of `toPandas()` to mimic dask for easily porting tests."""
         return self.toPandas()
 
+    @derived_from(pd.DataFrame)
```

Comment:
This is only going to work with dataframes (and not series), even if it is currently in `Frame`. Let's move this function in `DataFrame` for now, because the implementation for rows is going to be quite different.

Reply:
Ok!